components/table/doc/index.en-US.md

```diff
@@ -135,7 +135,6 @@ Sort property
 | `[nzSort]` | Sort direction | `'ascend' | 'descend' | null` | - |
 | `[nzSortKey]` | Key to sort by | `string` | - |
 | `(nzSortChange)` | Sort direction change callback | `EventEmitter<'ascend' \| 'descend' \| null>` | - |
-| `(nzSortChangeWithKey)` | Sort direction change callback with `key` | `EventEmitter<{key: string, value: 'ascend' \| 'descend' \| null}>` | - |
```

Comment:
Oops, Apply this suggestion caused the [Semantic Pull Request](https://github.com/zeke/semantic-pull-requests) check is fail.

Please change this commit message(`"Update components/table/doc/index.en-US.md"`), or rebase it. Thank you very much.